tests/wdio/application-loading-overlay/application-loading-overlay-spec.js

```diff
@@ -2,6 +2,7 @@ Terra.describeViewports('ApplicationLoadingOverlay', ['large'], () => {
   describe('Loading overlay management', () => {
     before(() => {
       browser.url('/raw/tests/terra-application/application-loading-overlay/loading-overlay-test');
+      browser.moveToObject('#root', 0, 0);
```

Comment:
Funnily, when running against the static site the cursor is not set to root 0,0, and causes this test to fail, this doesn't happen when running against the serve static service.... i don't know why but this should fix it for both local and travis builds.